github/handlers/ExecuteBlockActionHandler.ts

```diff
@@ -287,79 +287,13 @@ export class ExecuteBlockActionHandler {
                 }
                 case ModalsEnum.SHARE_PROFILE_PARAMS : {
                     const profileInteractionData = context.getInteractionData().value;
-                    const datAny = profileInteractionData as any;
+                    const dataAny = profileInteractionData as any;
                     const storeData = {
-                        profileParams : datAny as string[]
+                        profileParams : dataAny as string[]
                     }
```

Comment:
@VipinDevelops  this seems a bit hacky, using any and then using string []

Reply:
Correct @samad-yar-khan  I fixed this thing